diesel/src/migrations/migration_error.rs

```diff
@@ -55,6 +55,7 @@ impl From<io::Error> for MigrationError {
 }
 
 #[derive(Debug, PartialEq)]
+#[cfg_attr(feature = "dev", allow(enum_variant_names))]
```

Comment:
Do you hold the opinions that the current names are better than what clippy wants, or is this just to avoid the API change?